packages/select/src/components/select/suggest.tsx

```diff
@@ -90,6 +91,16 @@ export class Suggest<T> extends React.PureComponent<ISuggestProps<T>, ISuggestSt
         queryList: (ref: QueryList<T> | null) => (this.queryList = ref),
     };
 
+    constructor(props: ISuggestProps<T>, context?: any) {
+        super(props, context);
+
+        // Initialize the state.
+        this.state = {
+            isOpen: false,
```

Comment:
looks like we lost some logic from line 77 on the left about `popoverProps.isOpen`